### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by dboltovskyi**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+ATF Test Scripts to check #[issue number]
```
Comment (LuxoftAKutsan) [inline .github/PULL_REQUEST_TEMPLATE.md]: @dboltovskyi  please add comment that issue in sdl_core repo

**Review by LuxoftAKutsan — APPROVED** [review]
Please add comments for issue number

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+This PR is **[ready / not ready]** for review.
+
+### Summary
+[Summary of PR changes]
+
+### ATF version
+[ATF version to use]
+
+### Changelog
+##### Breaking Changes
```
Comment (GetmanetsIrina) [inline .github/PULL_REQUEST_TEMPLATE.md]: in my opinion "Breaking Changes" is not applicable for scripts, so can be removed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* [Breaking change info]
+
+##### Enhancements
+* [Enhancement info]
+
+##### Bug Fixes
+* [Bug Fix Info]
+
+### Tasks Remaining:
```
Comment (GetmanetsIrina) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe "Tasks Remaining" need to rename ,better to use "Dependencies" to paste info about related tasks.

**Review by GetmanetsIrina — APPROVED** [review]

**Review by aderiabin — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dboltovskyi) [reply .github/PULL_REQUEST_TEMPLATE.md]: @LuxoftAKutsan Corrected in d1e0828

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dboltovskyi) [reply .github/PULL_REQUEST_TEMPLATE.md]: @GetmanetsIrina Removed in d1e0828

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (dboltovskyi) [reply .github/PULL_REQUEST_TEMPLATE.md]: @GetmanetsIrina Renamed in d1e0828

**Comment by dboltovskyi** [comment]
Since review is finished this PR is closed
New one will be created into smartdevicelink repository

**Pull request closed without merging**